main.tf

```diff
@@ -141,6 +141,10 @@ locals {
   ]
 }
 
+data "aws_ecs_task_definition" "task" {
+  task_definition = aws_ecs_task_definition.task.family
+}
+
```

Comment:
Please move this to data.tf and it should be good

Reply:
Done. Thanks!